### PR TITLE
enable guac in openshift

### PR DESCRIPTION
**Pull request by lulf**
- feat: enable guac in openshift template
- feat: enable guac in app-sre

### CODE REVIEW

File: deploy/openshift/template.yaml [hunk 1]
```diff
@@ -32,7 +101,7 @@ objects:
 
         gql-listen-port: 8080
 
-        gql-debug: false
+        gql-debug: true
```
Comment (dejanb) [inline deploy/openshift/template.yaml:104]: Do we want to enable this by default at this point?

File: deploy/openshift/values.yaml [hunk 2]
```diff
@@ -83,9 +84,9 @@ vexination:
 keycloak:
   enabled: false
 guac:
-  enabled: false
+  enabled: true
   graphql:
-    debug: false
+    debug: true
```
Comment (dejanb) [inline deploy/openshift/values.yaml:89]: Same here

Comment (lulf) [reply deploy/openshift/values.yaml]: Fixed

Comment (dejanb) [reply deploy/openshift/values.yaml]: This one seems to still be 'true'

Comment (dejanb) [reply deploy/openshift/values.yaml]: OK. Looks good now

**Review by dejanb — APPROVED** [review]

**Pull request merged**